client/src/main/scala/skuber/api/client/impl/KubernetesClientImpl.scala

```diff
@@ -42,6 +42,7 @@ class KubernetesClientImpl private[client] (val requestMaker: (Uri, HttpMethod)
   val watchContinuouslyRequestTimeout: Duration,
   val watchContinuouslyIdleTimeout: Duration,
   val watchPoolIdleTimeout: Duration,
+  val defaultPoolSettings: ConnectionPoolSettings,
```

Comment:
That's a breaking change,
please add a default value
```
val defaultPoolSettings: ConnectionPoolSettings = ConnectionPoolSettings(actorSystem)
```

Reply:
This constructor has package protected visibility. Are you sure that you want to have a default value in this place? IMO it can be confusing that this default will be on many levels - I added it on `KubernetesClientImpl.apply` which is public after your suggestion below. WDYT?